src/domain/session/room/UnknownRoomViewModel.js

```diff
@@ -53,6 +61,38 @@ export class UnknownRoomViewModel extends ViewModel {
     }
 
     get kind() {
-        return "unknown";
+        return this._peekable ? "peekable" : "unknown";
+    }
+
+    get timelineViewModel() {
+        return this._timelineVM;
+    }
+
+    avatarUrl(size) {
+        return getAvatarHttpUrl(this._room.avatarUrl, size, this.platform, this._room.mediaRepository);
+    }
+
+    async peek() {
```

Comment:
We could rename this to `load()` so that it's consistent with `RoomViewModel`, where the `load()` method renders the timeline.